fe/fe-core/src/main/java/org/apache/doris/analysis/TableName.java

```diff
@@ -189,6 +205,18 @@ public String toSql() {
         return stringBuilder.toString();
     }
 
+    /**
+     * if this.field is empty, we think they are like,otherwise they must equal to other's
+     *
+     * @param other
+     * @return
+     */
+    public boolean like(TableName other) {
```

Comment:
should explain the like does not conform to the commutative law. and notice user that if want to get expected result which TableName should be parameter